packages/pwa-kit-extension-sdk/src/types.ts

```diff
@@ -36,3 +36,12 @@ export type BuildCandidatePathsOptions = {
     projectDir: string
     extensionEntries: ApplicationExtensionEntry[]
 }
+
+/**
+ * Represents an overridable import
+ */
```

Comment:
Could you add an example that would better help other devs understand what this is. The name itself wasn't representative of what I thought the type was, but I can't think of anything better. Maybe OverridableFile is more descriptive?

```
export interface OverridableFile {
    sourceImport: string
    path: string
    sourceExtension?: string
}
```

in this example I'm changing the focus to be on the file and not the import.. so we are saying that file X is overrideable, here is its "path" and this is the "sourceImport" and "sourceExtension" that is was found in. 

Not sure if that is what we want, but thats where my brain went with this.